Week1/homework/app.js

```diff
@@ -3,9 +3,162 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'moby_dick',
+    'the_adventures_of_huckleberry_finn',
+    'alices_adventures_in_wonderland',
+    'charlie_and_the_chocolate_factory',
+    'wuthering_heights',
+    'war_and_peace',
+    'anna_karenina',
+    'animal_farm',
+    'a_tale_of_two_cities',
+    'the_alchemist'
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  const Books = {
+    moby_dick: {
+      title: 'Moby-Dick',
+      author: 'Herman Melville',
+      language: 'English',
+      length: 720,
+      info:
+        'In Herman Melvilles Moby- Dick, Captain Ahab is an eerily compelling madman who focuses his distilled hatred and suffering(and that of generations before him) into the pursuit of a creature as vast, dangerous and unknowable as the sea itself.More than just a novel of adventure, this is a haunting social commentary populated with some of the most enduring characters in literature.Written with wonderfully redemptive humour, Moby Dick is a profound and timeless inquiry into character, faith and the nature of perception.'
```

Comment:
I would like to appreciate that you followed the instructions regarding naming conventions of the properties of the books object. I would like to add one thing here though.

ES6 introduced a new way of dealing with strings called [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). It makes use of back-ticks(`) and a couple of advantages of using them over plain strings are:
1. You can make use of variable names inside the strings which you can't do if you are using single or double quotes.
2. You can split really long lines as in the case of our `info` property into multi-line strings which are then easier to read.

I would suggest you give it a read. 🙂

Reply:
I've searched template strings and learn to use multine line  and many others,  thanks for the suggestion.